base/dict.jl

```diff
@@ -95,7 +98,7 @@ function Dict{K,V}(ps::Pair...) where V where K
     return h
 end
 # Note the constructors of WeakKeyDict mirror these here, keep in sync.
-Dict() = Dict{Any,Any}()
+Dict(;sizehint...) = Dict{Any,Any}(;sizehint...)
```

Comment:
```suggestion
Dict(;args...) = Dict{Any,Any}(;args...)
```
I know, there is only single argument right now, but `args` seems to be more commonly used for this case.

Reply:
I think this provides useful implicit documentation that is lost if we replace `sizehint...` with `args...` or [`kws...`](https://github.com/JuliaLang/julia/pull/46225)
```
julia> methods(Dict)
# 5 methods for type constructor:
[1] Dict(;sizehint...) in Base at dict.jl:118
[2] Dict(ps::Pair{K, V}...) where {K, V} in Base at dict.jl:124
[3] Dict(kv::Tuple{}) in Base at dict.jl:119
[4] Dict(ps::Pair...) in Base at dict.jl:125
[5] Dict(kv) in Base at dict.jl:127
```

Reply:
```suggestion
Dict(; sizehint = 0) = Dict{Any,Any}(; sizehint)
```